Api.Flickr/Abstractions/IFlickrClient.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using Api.Flickr.Models;
+
+namespace Api.Flickr.Abstractions
```

Comment:
Its not huge mistake, but why dont call folder Interfaces?

Reply:
I wanted to call it 'Abstractions'. This is my usual trick.